src/Grid/GridHeader.js

```diff
@@ -182,6 +182,41 @@ class GridHeader extends React.Component {
     });
 
     this.setState({ sortOrder }, this.state.dataSource.sort(this.state.rowsPerPage, this.state.page) );
+  } */
+
+  multiSort = (event, property) => {
+    const array = Object.assign({}, this.state.dataSource);
+    
+    array.columns.forEach( (column, i) => {
+      if(column.Name === property){
+        column.SortDirection = column.SortDirection === 'Ascending' ? 
```

Comment:
Again, this logic can be more readable, check https://github.com/unosquare/tubular/blob/74d0eee2735dc0b966a4dca893974fa895414a75/src/js/tubular/directives/grid/tbGridCtrl.js#L434